### PR TITLE
AC Automated Test - LRAC-9126 | master

### PR DESCRIPTION
**Pull request by rafaellajordao**
https://issues.liferay.com/browse/LRAC-9126

### CODE REVIEW

**Comment by rafaellajordao** [comment]
ci:test:sf

**Comment by christianstokes** [comment]
looks good! cc @alexcandidos

**Comment by christianstokes** [comment]
@rafaellajordao actually can you add some "task" descriptions to this one as well, even though it is short? Thank you!

**Comment by rafaellajordao** [comment]
ci:test:sf

**Comment by christianstokes** [comment]
LGTM!

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/NewProperty.testcase [hunk 1]
```diff
@@ -30,31 +24,26 @@ definition {
...
-	}
+	test CantCreateAPropertyWithNameInBlankOrNull {
+		task ("Go to properties page, try to add a new unnamed property and assert that the save button is disabled") {
+			ACNavigation.goToSettings();
 
-	@description = "Feature ID: Legacy | Automation ID: LRAC-9125 | Test Summary: Property with its name edited"
-	@ignore = "true"
-	@priority = "5"
-	test CanEditedPropertyName {
+			ACSettings.goToProperties();
```
Comment (alexcandidos) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/NewProperty.testcase:33]: hi @rafaellajordao , the idea of having tasks is to group actions (navigations, assertions, etc) in a way that will allow the reader to understand better what is happening in the test case steps.

In this case, even with a simple case, we would have at least 3 tasks:

Task 1 (describing the navigation): Go to Properties screen under AC Settings
Task 2 (additional actions): Add a new property with an empty name value
Task 3 (assertions): Assert that is not possible to save when the property name is empty


What do you think?

Thanks! :)

Comment (rafaellajordao) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/NewProperty.testcase]: thanks @alexcandidos!! fixed!! :)

**Comment by rafaellajordao** [comment]
ci:test:sf

**Comment by alexcandidos** [comment]
Hi @rafaellajordao , Thanks for working on it!

The code looks good to me!

I'm forwarding it.

**Comment by alexcandidos** [comment]
ci:forward

**Pull request closed without merging**